x/auth/client/rest.go

```diff
@@ -51,7 +51,8 @@ func WriteGenerateStdTxResponse(w http.ResponseWriter, clientCtx client.Context,
 		return
 	}
 
-	output, err := clientCtx.JSONMarshaler.MarshalJSON(types.NewStdTx(stdMsg.Msgs, stdMsg.Fee, nil, stdMsg.Memo))
+	// NOTE: amino is used intentionally here, don't migrate it
+	output, err := clientCtx.Codec.MarshalJSON(types.NewStdTx(stdMsg.Msgs, stdMsg.Fee, nil, stdMsg.Memo))
```

Comment:
Could you explain what's the difference between `clientCtx.Codec.MarshalJSON` vs `clientCtx.JSONMarshaler.MarshalJSON`?

Reply:
Using `Codec` ensures that we are using amino (it should be renamed to `LegacyAminoCodec` or something eventually). `JSONMarshaler` could be proto or amino and we only want amino here.